examples/userapi/05_conditions.ipynb

```diff
@@ -0,0 +1,489 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ConditionalDimension tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is designed to demonstrate how users can implement various conditions in Devito using [ Conditional Dimension](https://github.com/devitocodes/devito/blob/e918e26757a4f0ea3c3c416b6e0f48f4e5023c37/devito/types/dimension.py#L635 \"class ConditionalDimension(DerivedDimension):\"). Devito can generate a great variety of loop kernels using its Domain Specific Language. We are going to show how we can generate loop structures in C language using Devito's DSL and how we can add conditional statements in them.\n",
```

Comment:
How about just writing the object name instead of a hyper link (here)? i.e. '... using `ConditionalDimension`s. I think hyperlinks to code are a good idea but probably best kept for some form of index at the end of the tutorial since people who are learning this stuff may well find being pointed to source code so early on confusing.